dali/python/nvidia/dali/pipeline.py

```diff
@@ -638,33 +671,29 @@ def _setup_pipe_pool_dependency(self):
             # The sole point of this call is to ensure the lifetime of the pool exceeds the lifetime
             # of the pipeline's backend, so that shared memory managed by the pool is not freed
             # before pipline's backend is garbage collected.
-            # Otherwise the backend may try to access unmmaped memory which leads to crashes at the Python teardown.
+            # Otherwise the backend may try to access unmmaped memory which leads to
+            # crashes at the Python teardown.
             self._pipe.SetPyObjDependency(self._py_pool)
 
     def _start_py_workers(self):
         if not self._parallel_input_callbacks:
             return
-        self._py_pool = WorkerPool.from_groups(
-            self._parallel_input_callbacks, self._prefetch_queue_depth, self._py_start_method,
-            self._py_num_workers, py_callback_pickler=self._py_callback_pickler)
+        self._py_pool = WorkerPool.from_groups(self._parallel_input_callbacks,
+                                               self._prefetch_queue_depth, self._py_start_method,
+                                               self._py_num_workers,
+                                               py_callback_pickler=self._py_callback_pickler)
```

Comment:
One more line break won't hurt here...
```suggestion
        self._py_pool = WorkerPool.from_groups(self._parallel_input_callbacks,
                                               self._prefetch_queue_depth,
                                               self._py_start_method,
                                               self._py_num_workers,
                                               py_callback_pickler=self._py_callback_pickler)
```